compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala

```diff
@@ -147,7 +152,12 @@ class PlainPrinter(_ctx: Context) extends Printer {
       case tp: TermRef if tp.denot.isOverloaded =>
         "<overloaded " ~ toTextRef(tp) ~ ">"
       case tp: TypeRef =>
-        toTextPrefix(tp.prefix) ~ selectionString(tp)
+        printWithoutPrefix.find(sym => tp.isDirectRef(sym)) match {
```

Comment:
I would prefer not to hard-code these. Can't we instead check when printing whether a class has an alias in `Predef` or `scala` with the same name?

Reply:
Possibly, but that would be more expensive and may require forcing things.